### PR TITLE
Bump version to 0.2.4

### PR DESCRIPTION
**Pull request by robertcv**

### CODE REVIEW

**Comment by PrimozGodec** [comment]
@conda-forge-admin, please rerender

**Comment by PrimozGodec** [comment]
@conda-forge-admin, please rerender

**Comment by robertcv** [comment]
@conda-forge-admin, please rerender

**Comment by robertcv** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,16 +12,17 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1000
-  script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: True  # [py2k]
+  number: 10
```
Comment (PrimozGodec) [inline recipe/meta.yaml]: Reset build number to 0

**Comment by PrimozGodec** [comment]
It looks good now. Please just reset the build number to 0 since it is first built for this package version.

**Comment by PrimozGodec** [comment]
Great. :) @astaric @kernc can one of you merge this PR?

**Comment by kernc** [comment]
It says:

> Only those with write access to this repository can merge pull requests.

:sweat_smile:

**Comment by PrimozGodec** [comment]
@conda-forge/core, can someone merge it?

**Comment by isuruf** [comment]
@conda-forge-admin, please update team

**Comment by isuruf** [comment]
@kernc, did you leave conda-forge github org?

**Pull request merged**